docs/guides/high-availability/deploy-aws-accelerator-fencing-lambda.adoc

```diff
@@ -2,38 +2,40 @@
 
 == Architecture
-In the event of a network communication failure between the two sites in a multi-site deployment, it is no
-longer possible for the two sites to continue to replicate data between themselves and the two sites
-will become increasingly out-of-sync. As it is possible for subsequent Keycloak requests to be routed to different
-sites, this may lead to unexpected behaviour as previous updates will not have been applied to both sites.
 
-In such scenarios a quorum is commonly used to determine which sites are marked as online or offline, however as multi-site deployments only consist of two sites, this is not possible.
-Instead, we leverage "`fencing`" to ensure that when one of the sites is unable to connect to the other site, only one site remains in the loadbalancer configuration and hence only this site is able to serve subsequent users requests.
+In the event of a network communication failure between the two sites in a multi-site deployment, it is no longer possible for the two sites to continue to replicate data between them.
+As this setup values consistency over availability, and as the {jdgserver_name} caches are configured with a failure policy `FAIL`, all user requests are served with an error message until this is resolved by either restoring the network connection or disabling the replication.
 
-Once the fencing procedure is triggered the replication between two {jdgserver_name} clusters in each site is no longer enabled and as a result the sites will be out-of-sync.
-To recover from the out-of-sync state a manual re-sync is necessary as described in <@links.ha id="operate-synchronize" />.
-This is why a site which is removed via fencing will not be re-added automatically when the network communication failure is resolved, but only after such a synchronisation using the manual procedure <@links.ha id="operate-site-online" />.
+In such scenarios, a quorum is commonly used to determine which sites are marked as online or offline.
+However, as multi-site deployments only consist of two sites, this is not possible.
+Instead, we leverage "`fencing`" to ensure that when one of the sites is unable to connect to the other site, only one site remains in the loadbalancer configuration, and hence only this site is able to serve subsequent users requests.
+
+In addition to the loadbalancer configuration, the fencing procedure disables replication between the two {jdgserver_name} clusters to allow serving user requests from the site that remains in the loadbalancer configuration.
+As a result, the sites will be out-of-sync once the replication has been disabled.
+
+To recover from the out-of-sync state, a manual re-sync is necessary as described in <@links.ha id="operate-synchronize" />.
+This is why a site which is removed via fencing will not be re-added automatically when the network communication failure is resolved, but only after such a synchronisation and using the manual procedure <@links.ha id="operate-site-online" />.
```

Comment:
"This is why a site which is removed via fencing will not be re-added automatically when the network communication failure is resolved. The remove site should only be re-added once the two sites have been synchronized using the outlined procedure <@links.ha id="operate-site-online" />."